samples/scales/time/financial.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html>
+
+<head>
+	<title>Line Chart</title>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js"></script>
+	<script src="../../../dist/Chart.js"></script>
```

Comment:
@benmccann can you use `Chart.bundle.js` instead and remove the `cdnjs.cloudflare.com` dependency, like that the example can be open locally / offline from the ZIP archive.

Reply:
That doesn't work unfortunately. It results in `Uncaught ReferenceError: moment is not defined`. These couple lines are in the other time samples as well